### PR TITLE
Feat/health insurance

### PR DESCRIPTION
**Pull request by acrepet**

### CODE REVIEW

File: backend/src/main/java/org/globe42/web/persons/PersonDTO.java [hunk 1]
```diff
@@ -149,6 +151,8 @@ public HealthCareCoverage getHealthCareCoverage() {
         return healthCareCoverage;
     }
 
+    public String getHealthInsurance() { return healthInsurance; }
```
Comment (jnizet) [inline backend/src/main/java/org/globe42/web/persons/PersonDTO.java]: Please, rebase on master, and use the same format for this getter (and the other ones) as for all the other ones (instructions of the method on a separate line):

```
public String getHealthInsurance() { 
    return healthInsurance; 
}
```

I guess you would like the code to be more concise, but I suggested using Kotlin :-) 

And to me that makes it inconsistent, and less readable.

**Comment by acrepet** [comment]
I have to rebase my branch!

**Comment by jnizet** [comment]
You're not forced to delete a PR to rebase your branch. You can just rebase it, then `git push --force-with-lease`. You know you have a git teacher at home, right ;-)

**Comment by acrepet** [comment]
OK, I've rebased on master and fix the getters

**Pull request merged**